### PR TITLE
Fixes #11342: Add link toward slack notification connector

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/11342

### CODE REVIEW

File: 23_configuration_management/36_compliance.txt [hunk 1]
```diff
@@ -5,28 +5,29 @@
 ==== Overview in Rudder
 
 
-Rudder is build to continuously assesses drift compared to defined policies, with or without auto-healing. By auto-healing, 
-we mean that optionally, Rudder can continuously enforces configuration over time, correcting the assessed drift so that 
-your configuration converge toward desired states (for how to configure policy mode by node, rule or directive, see <<_policy_mode_audit_enforce>>).
+Rudder is build to continuously assesses drift compared to defined policies, with or without auto-healing.
```
Comment (amousset) [inline 23_configuration_management/36_compliance.txt]: built

Comment (amousset) [reply 23_configuration_management/36_compliance.txt]: assess

File: 23_configuration_management/36_compliance.txt [hunk 2]
```diff
@@ -5,28 +5,29 @@
 ==== Overview in Rudder
 
 
-Rudder is build to continuously assesses drift compared to defined policies, with or without auto-healing. By auto-healing, 
-we mean that optionally, Rudder can continuously enforces configuration over time, correcting the assessed drift so that 
-your configuration converge toward desired states (for how to configure policy mode by node, rule or directive, see <<_policy_mode_audit_enforce>>).
+Rudder is build to continuously assesses drift compared to defined policies, with or without auto-healing. 
+
+By auto-healing, we mean that optionally, Rudder can continuously enforces correct configuration over time, correcting the assessed drift so that
```
Comment (amousset) [inline 23_configuration_management/36_compliance.txt]: enforce

File: 23_configuration_management/36_compliance.txt [hunk 3]
```diff
@@ -5,28 +5,29 @@
 ==== Overview in Rudder
 
 
-Rudder is build to continuously assesses drift compared to defined policies, with or without auto-healing. By auto-healing, 
-we mean that optionally, Rudder can continuously enforces configuration over time, correcting the assessed drift so that 
-your configuration converge toward desired states (for how to configure policy mode by node, rule or directive, see <<_policy_mode_audit_enforce>>).
+Rudder is build to continuously assesses drift compared to defined policies, with or without auto-healing. 
+
+By auto-healing, we mean that optionally, Rudder can continuously enforces correct configuration over time, correcting the assessed drift so that 
+your configuration converge toward desired states. This behavior is optionnal, and Rudder can only report drift without changing configuration.
```
Comment (amousset) [inline 23_configuration_management/36_compliance.txt]: converges

Comment (amousset) [inline 23_configuration_management/36_compliance.txt]: towards

**Comment by fanf** [comment]
PR rebased

**Pull request merged**